contrib/babelfishpg_common/sql/binary.sql

```diff
@@ -91,21 +91,21 @@ CREATE CAST (sys.BPCHAR AS sys.BBF_BINARY)
 WITH FUNCTION sys.bpcharbinary (sys.BPCHAR, integer, boolean) AS ASSIGNMENT;
 
 
-CREATE OR REPLACE FUNCTION sys.binarysysvarchar(sys.BBF_BINARY)
+CREATE OR REPLACE FUNCTION sys.binarysysvarchar(sys.BBF_BINARY, integer, boolean)
 RETURNS sys.VARCHAR
 AS 'babelfishpg_common', 'varbinaryvarchar'
 LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 
 CREATE CAST (sys.BBF_BINARY AS sys.VARCHAR)
-WITH FUNCTION sys.binarysysvarchar (sys.BBF_BINARY) AS ASSIGNMENT;
+WITH FUNCTION sys.binarysysvarchar (sys.BBF_BINARY, integer, boolean) AS ASSIGNMENT;
```

Comment:
Lets have some test cases which uses these cast in prepare script and then we try to upgrade it,

Prepare:
```
create view some_view as select cast(cast(some_value as binary) as varchar)
```

verify --
```
extract the view definition, SELECT pg_catalog.pg_get_viewdef(oid, true) FROM pg_class WHERE relname = 'some_view';
```